app/controllers/application_controller/ci_processing.rb

```diff
@@ -400,7 +400,7 @@ def resize_vm
     load_edit("vm_resize__#{params[:id]}")
     flavor_id = @edit[:new][:flavor]
     flavor = find_by_id_filtered(Flavor, flavor_id)
-    @record = VmOrTemplate.find_by_id(params[:id])
+    @record = find_by_id_filtered(VmOrTemplate, params[:id])
```

Comment:
`find_by_id_filtered` method was renamed to `find_record_with_rbac` (https://github.com/ManageIQ/manageiq-ui-classic/pull/779/commits/dad34557eb21d21317126a51f30f9384197c0ea4)